cmd/sync.go

```diff
@@ -13,6 +15,10 @@ var syncCmd = &cobra.Command{
 }
 
 func syncMarketData(cmd *cobra.Command, args []string) {
+	if !viper.GetBool("market.enabled") {
```

Comment:
I think we should check this in higher function, not inside

Reply:
inside `root.go`? How do we check this?